plugin/core/windows.py

```diff
@@ -739,3 +756,145 @@ def _format_request(self, direction: str, method: str, request_id: Any) -> str:
 
     def _format_notification(self, direction: str, method: str) -> str:
         return "{} {} {}".format(direction, self._server_name, method)
+
+
+class RemoteLogger(Logger):
+    PORT = 9981
+    DIRECTION_OUTGOING = 1
+    DIRECTION_INCOMING = 2
+    _ws_server = None  # type: Optional[WebsocketServer]
+    _ws_server_thread = None  # type: Optional[threading.Thread]
+
+    def __init__(self, manager: WindowManager, server_name: str) -> None:
+        self._manager = ref(manager)
```

Comment:
Where is this member variable used?

Reply:
It probably isn't.

Reply:
Removed.

(That also reminded me that I need to add handling for logging server's stderr.